.github/workflows/workflow.yml

```diff
@@ -2,11 +2,6 @@ on:
   push:
     branches:
       - main
-    paths:
```

Comment:
This section is used to make the action trigger only on changes to certain files. It's not required. In my example repo did it mainly to keep it from deploying on changes to the README. You can remove it completely like this or update it to include the things required for the python app (e.g. `server.py`).

https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths